src/compiler/CompiledOperation.ts

```diff
@@ -60,6 +60,10 @@ export default class CompiledOperation {
     }, {});
   }
 
+  public getDefinedBodyContentType() {
```

Comment:
Is it necessary to add some doco that this is only for request bodies?

Reply:
I can rename the function to make it more clear.